include/modem/lte_lc.h

```diff
@@ -1189,6 +1199,23 @@ int lte_lc_periodic_search_clear(void);
  */
 int lte_lc_periodic_search_request(void);
 
+/** @brief Read current reduced mobility mode.
+ *
```

Comment:
Could add a note here stating from which modem fw version this AT command is available.